### PR TITLE
Work section

### PR DESCRIPTION
**Pull request by kusiLaw**
This pull request adds works and about section myself to the portfolio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,27 +29,255 @@
...
+                    </div>
+                    <button type="button" class="secondary">Get my resume</button>
+                </div>
+
+                <ul class="skill-list">
+                    <li class="frame12">
+                        <div class="frame7">
+                            <!-- <img class="btn-separator" src="./images/Separator-Bottom.png" alt=""> -->
+                            <span>Languages</span>
+                            <button type="button" class="normal-btn"> &#8964; </button>
```
Comment (vechicin) [inline index.html:239]: The icons' size doesn't match the template design. I recommend you download the icons directly from the Figma template and apply them to this section.

File: index.html [hunk 2]
```diff
@@ -25,27 +29,255 @@
...
                                     <li><a href="#"><img src="./images/like.svg" alt="like"></a></li>
                                 </ul>
                             </div>
                         </div>
                     </div>
-                </div>    
+                </div>
             </section>
+
+            <section class="work">
```
Comment (vechicin) [inline index.html:55]: The margins for this section are not quite right. The content should be centered and it is slightly slanted to the left. Please fix it.

Screen Shot 2022-07-07 at 9 04 45

Screen Shot 2022-07-07 at 9 05 05

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @kusiLaw ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (kusiLaw) [reply index.html:239]: thank you very much, I have been trying to export but anytime I export the icon from figma it is empty.
will it be allowed, if I increase its font size to 24px to have the same sizes as the one on figma?.
 thank you

**Pull request merged**